### PR TITLE
Local Server Map Image Layer

### PR DESCRIPTION
**Pull request by nCastle1**
This sample demonstrates how to load a map image layer from a local map service.

### CODE REVIEW

**Comment by nCastle1** [comment]
@ThadT Can you please give this a review if you have time? Thanks!

File: src/WPF/ArcGISRuntime.WPF.Samples/Samples/LocalServer/LocalServerMapImageLayer/metadata.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "Name": "Local Server Map Image Layer",
+  "SampleName": "LocalServerMapImageLayer",
+  "Description": "This sample demonstrates how to display a Map Image Layer from a local map service",
+  "Instructions": "This sample depends on the local server being installed and configured. See https://developers.arcgis.com/net/latest/wpf/guide/local-server.htm for details and instructions.\nSample date is loaded in the background. ",
```
Comment (ThadT) [inline src/WPF/ArcGISRuntime.WPF.Samples/Samples/LocalServer/LocalServerMapImageLayer/metadata.json]: "Sample date ..." -> "Sample **data** ..."

File: src/WPF/ArcGISRuntime.WPF.Samples/Samples/LocalServer/LocalServerMapImageLayer/LocalServerMapImageLayer.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    public partial class LocalServerMapImageLayer
+    {
+        // Hold a reference to the local map service
+        private LocalMapService _localMapService;
+
+        public LocalServerMapImageLayer()
+        {
+            InitializeComponent();
+
+            // Create the UI, setup the control references and execute initialization
```
Comment (ThadT) [inline src/WPF/ArcGISRuntime.WPF.Samples/Samples/LocalServer/LocalServerMapImageLayer/LocalServerMapImageLayer.xaml.cs]: This comment seems to get copied to every sample, but isn't always accurate. In this case, no UI is being created, e.g.

File: src/WPF/ArcGISRuntime.WPF.Samples/Samples/LocalServer/LocalServerMapImageLayer/LocalServerMapImageLayer.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+                // Be notified when the map service is loaded
+                _localMapService.StatusChanged += _localMapService_StatusChanged;
+
+                // Start the feature service
+                await _localMapService.StartAsync();
+            }
+            catch (Exception ex)
+            {
+                MessageBox.Show(ex.Message, "Local Server failed to start");
+            }
```
Comment (ThadT) [inline src/WPF/ArcGISRuntime.WPF.Samples/Samples/LocalServer/LocalServerMapImageLayer/LocalServerMapImageLayer.xaml.cs]: Need a new line after `try/catch` block.

**Comment by nCastle1** [comment]
@ThadT Thanks for the review! I've made all changes and will now merge.

**Pull request merged**